modules/modules.nix

```
@@ -128,6 +128,7 @@ let
     (loadModule ./services/screen-locker.nix { })
     (loadModule ./services/stalonetray.nix { })
     (loadModule ./services/status-notifier-watcher.nix { })
+    (loadModule ./services/sxhkd.nix { })
```

Since this module uses systemd it should be marked as Linux only:
```suggestion
    (loadModule ./services/sxhkd.nix { condition = hostPlatform.isLinux; })
```